package.json

```diff
@@ -26,12 +26,13 @@
   },
   "homepage": "https://github.com/pawelgalazka/runjs#readme",
   "dependencies": {
-    "babel": "5.8.23",
     "chalk": "1.1.0",
     "chokidar": "1.2.0",
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
+    "pkg-config": "^1.1.0"
   },
   "devDependencies": {
+    "babel": "5.8.23",
```

Comment:
Yup I totally agree, but I still thinking if it's possible to have a final fallback to the babel which is within runjs repository.